app/build.gradle.kts

```diff
@@ -33,10 +33,17 @@ android {
         }
     }
     buildTypes {
-        debug { isMinifyEnabled = false }
-        release {
-            isMinifyEnabled = true
-            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        // for local builds, used to find shrinking issues
+        val isMinify = project.hasProperty("minify")
+        if (isMinify) {
+            debug {
+                // while isDebuggable is set to true no obfuscation takes place,
+                // the shrinking phase will still remove unused classes
+                isDebuggable = true
```

Comment:
basically true is the default value here, it was added here basically just to make the comment more clear.

When playing around I was wondering why the classes were not obfuscated when adding this. I tested the behavior by adding an "UnusedClass" to the SDK. Since shrinking is disabled in the SDK it was not removed from the corresponding build aar. However when building the TestApp with the minify parameter it was correctly removed during shrinking when it was unused from inside the TestApp. When the TestApp was using the class, it was correctly left intact